pkg/csls/stream_test.go

```diff
@@ -12,6 +16,17 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func gzip_decompress(b []byte) ([]byte, error) {
```

Comment:
nit: casing is funny for a go program

```suggestion
func gzipDecompress(b []byte) ([]byte, error) {
```